configure.ac

```diff
@@ -768,10 +781,10 @@ if test "$CQL_CONFIG" != "" ; then
         AC_MSG_ERROR([--with-cql should point to a cql_config program])
     fi
 
-    CQL_INCLUDEDIR=`$CQL_CONFIG --includedir`
-    CQL_CPPFLAGS="`$CQL_CONFIG --cppflags` -I$CQL_INCLUDEDIR"
-    CQL_LIBS="-L`$CQL_CONFIG --libdir` -lcassandra -luv"
-    CQL_VERSION=`$CQL_CONFIG --version`
+    CQL_INCLUDEDIR=`$CQL_CONFIG --cflags-only-I $cql_lib`
```

Comment:
Need space between cflags-only and -I

Reply:
No,
there is a different option for --cflags. 
See --help
This is a separate option:
--cflags-only-I                         output -I flags
The output gives the flags with the -I prefix included.